gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```

Comment:
Windows Server R2. Seems like `$APP_HOME/gradle/wrapper/gradle-wrapper.jar` is missed.

```
d:\tmp\screemshotTaker\screenshotTaker>gradlew.bat integrationTest -PdeviceSerial=emulator-5554
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain

d:\tmp\screemshotTaker\screenshotTaker>gradlew.bat help
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain                
```